ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/BaseFeePendingTransactionsTest.java

```diff
@@ -125,21 +134,34 @@ public void shouldGetTransactionByHash() {
   public void shouldDropOldestTransactionWhenLimitExceeded() {
     final Transaction oldestTransaction =
         transactionWithNonceSenderAndGasPrice(0, SIGNATURE_ALGORITHM.get().generateKeyPair(), 10L);
-    transactions.addRemoteTransaction(oldestTransaction);
+    senderLimitedTransactions.addRemoteTransaction(oldestTransaction);
```

Comment:
small thing i wanted to call out. This is a much better variable name! It's telling me what it is and _why it is important_ instead of just reproducing the class name.